app/components/forms/orders/order-form.js

```diff
@@ -14,14 +14,26 @@ import {
 export default Component.extend(FormMixin, {
   router: service(),
 
+  buyerFirstName: computed('buyerHasFirstName', function() {
```

Comment:
Never mutate a plain computed property with only a getter directly. That is an incorrect usage. Use [`oneWay`](https://api.emberjs.com/ember/3.10/functions/@ember%2Fobject%2Fcomputed/oneWay) here instead. That exists exactly for this purpose.

Reply:
@niranjan94 If I am understanding your concern correctly, I should use it in this way, right?
```js
buyerFirstName: oneway('buyerHasFirstName')
```

Reply:
@uds5501 No ,
```
 buyerFirstName: computed.oneWay('buyerHasFirstName')
```